AK/Optional.h

```diff
@@ -261,6 +288,26 @@ requires(!IsLvalueReference<T>) class [[nodiscard]] Optional<T> : public Optiona
         return *this;
     }
 
+    template<class U = T>
+    requires(
+        !OneOf<RemoveCVReference<U>, Optional, OptionalNone>
```

Comment:
In this position, please use `IsOneOf<RemoveCVReference<U>, Optional, OptionalNone>`.
I would also prefer if you spelled out the name `Optional<T>` (but it's not a necessity)

Reply:
I made it `!IsOneOfIgnoringCVReference<U, Optional<T>, OptionalNone>`